tests/schema/tests.py

```diff
@@ -2298,3 +2315,16 @@ class Meta:
         doc = Document.objects.create(name='Test Name')
         student = Student.objects.create(name='Some man')
         doc.students.add(student)
+
+    @skipIfDBFeature('supports_index_on_text_field')
+    def test_db_feature_supports_index_on_text_field(self):
+        class NoteInfoTextFieldIndexed(Model):
```

Comment:
There is no need to declare another model, please use `NoteInfoIndexed` like in the previous test.